### PR TITLE
jotix layouts

### PR DESCRIPTION
**Pull request by jotix**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/community/ortho_4x4/jotix/keymap.c [hunk 1]
```diff
@@ -1,41 +1,82 @@
...
+
+[_FN] = LAYOUT_ortho_4x4 (
+  KC_PERC,KC_CIRC,KC_AMPR,KC_ASTR,
+  KC_EXLM,KC_AT,  KC_HASH,KC_DLR,
+  KC_Z,   KC_X,   KC_C,   _______,
+  KC_NLCK,_______,_______,TGGAME
+)
 };
+
+uint32_t layer_state_set_user(uint32_t state) {
```
Comment (drashna) [inline layouts/community/ortho_4x4/jotix/keymap.c:39]:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```

File: layouts/community/ortho_4x4/jotix/keymap.c [hunk 2]
```diff
@@ -1,41 +1,82 @@
...
+  #ifdef JOTPAD16_LEDS
+  if (biton32(state) == _FN) {
```
Comment (drashna) [inline layouts/community/ortho_4x4/jotix/keymap.c:41]:
```suggestion
  if (get_highest_layer(state) == _FN) {
```

File: layouts/community/ortho_4x4/jotix/keymap.c [hunk 3]
```diff
@@ -1,41 +1,82 @@
...
+    writePinHigh(JOTPAD16_LED1);
+  } else {
+    writePinLow(JOTPAD16_LED1);
+  }
+  if (biton32(state) == _GAMEPAD) {
```
Comment (drashna) [inline layouts/community/ortho_4x4/jotix/keymap.c:46]:
```suggestion
  if (get_highest_layer(state) == _GAMEPAD) {
```

File: layouts/community/ortho_4x4/jotix/keymap.c [hunk 4]
```diff
@@ -1,41 +1,82 @@
...
+    writePinHigh(JOTPAD16_LED2);
+  } else {
+    writePinLow(JOTPAD16_LED2);
+  }
+  #endif
+  return state;
+}
```
Comment (drashna) [inline layouts/community/ortho_4x4/jotix/keymap.c:53]: Actually, this entire thing could be written as:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
#ifdef JOTPAD16_LEDS
    writePin(JOTPAD16_LED1, (get_highest_layer(state) == _FN));
    writePin(JOTPAD16_LED2, (get_highest_layer(state) == _GAMEPAD));
#endif
  return state;
}
```

File: layouts/community/ortho_4x4/jotix/keymap.c [hunk 5]
```diff
@@ -1,41 +1,82 @@
...
+
+bool led_update_user(led_t led_state) {
+  // NumLock allways on
+  if (!led_state.num_lock) {
+    tap_code(KC_NUMLOCK);
+  }
+  return true;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```
Comment (drashna) [inline layouts/community/ortho_4x4/jotix/keymap.c:63]: Nice.  Though, you could also do the same with the "combo" feature.  But for just the one function ... that may be overkill.

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**